source/client-side-encryption/client-side-encryption.rst

```diff
@@ -1938,9 +1969,10 @@ little as possible.
 Why aren't we creating a unique index in the key vault collection?
 ------------------------------------------------------------------
 
-There should be a unique index on keyAltNames. Although GridFS
+There should be a unique index on the ``keyAltNames`` field with a partial index
+filter for only documents where ``keyAltNames`` exists. Although GridFS
 automatically creates indexes as a convenience upon first write, it has
-been problematic before. It requires the createIndex privilege, which a
+been problematic before due to requiring the ``createIndex`` privilege, which a
```

Comment:
On the off chance you want to link this: https://www.mongodb.com/docs/manual/reference/privilege-actions/#mongodb-authaction-createIndex

No worries if not.